frontend/device/kindle/device.lua

```diff
@@ -354,6 +353,17 @@ function KindleOasis:init()
 
     self.input.open(self.touch_dev)
     self.input.open("/dev/input/by-path/platform-gpiokey.0-event")
+
+    -- get rotate dev by EV=d
+    local std_out = io.popen("cat /proc/bus/input/devices | grep -e 'Handlers\\|EV=' | grep -B1 'EV=d'| grep -o 'event[0-9]'", "r")
```

Comment:
[root@kindle root]# ls /dev/input/by-path/
platform-gpiokey.0-event
platform-imx-i2c.0-platform-max77696-onkey.0-event
platform-imx-i2c.1-event


First one is page button input,

Not sure the second one

[root@kindle root]# evtest /dev/input/by-path/platform-imx-i2c.0-platform-max776
96-onkey.0-event 
Input driver version is 1.0.1
Input device ID: bus 0x0 vendor 0x0 product 0x0 version 0x0
Input device name: "max77696-onkey"
Supported events:
  Event type 0 (Sync)
  Event type 1 (Key)
    Event code 116 (Power)
Testing ... (interrupt to exit)

The third is the touch screen.